.evg.yml

```diff
@@ -324,174 +405,339 @@ tasks:
+    post:
+      - command: attach.xunit_results
+        params:
+        files:
+          - "./stitch-java-sdk/android/services/mongodb-remote/build/outputs/androidTest-results/connected/*.xml"
+
+  - name: run_core_services_twilio_tests
+    commands:
+      - command: shell.exec
+        params:
+          shell: "bash"
+          script: |
+            set -e
+            sh stitch-java-sdk/etc/run_core_test.sh :core:core-services:stitch-core-services-twilio:test
+    post:
+      - command: attach.xunit_results
+        params:
+        files:
+          - "./stitch-java-sdk/core/services/twilio/build/test-results/test/*.xml"
+
+  - name: run_android_services_twilio_tests
+    commands:
+      - command: shell.exec
+        params:
+          shell: "bash"
+          script: |
+            set -e
+            sh stitch-java-sdk/etc/run_core_test.sh :android:android-services:stitch-android-services-twilio:connectedAndroidTest
+    post:
+      - command: attach.xunit_results
+        params:
+        files:
+          - "./stitch-java-sdk/android/services/twilio/build/outputs/androidTest-results/test/*.xml"
+
+task_groups:
+  - name: run_core_tests
+    setup_group_can_fail_task: true
+    max_hosts: 2
+    setup_group:
+      - func: "fetch_source"
+      - func: "setup_android"
+    tasks:
+      - run_core_sdk_tests
+      - run_core_services_mongodb_remote_tests
+      - run_core_services_aws_tests
+      - run_core_services_aws_s3_tests
+      - run_core_services_aws_ses_tests
+      - run_core_services_fcm_tests
+      - run_core_services_http_tests
+      - run_core_services_mongodb_local_tests
+      - run_core_services_twilio_tests
+  - name: run_android_tests
+    setup_group_can_fail_task: true
+    max_hosts: 4
+    setup_group:
+      - func: "fetch_source"
+      - func: "setup_mongod"
+      - func: "setup_android"
+      - func: "run_mongod"
+      - func: "setup_stitch"
+      - func: "setup_test_creds"
+      - func: "setup_android_sdk_env_vars"
+      - func: "cleanup_adb"
+      - func: "create_emulator"
+      - func: "setup_logcat"
+    setup_task:
+      - func: "run_mongod"
+      - func: "run_stitch"
+      - func: "setup_emulator"
+    tasks:
+      - run_android_sdk_tests
+      - run_android_services_mongodb_remote_tests
+      - run_android_services_aws_tests
+      - run_android_services_aws_s3_tests
+      - run_android_services_aws_ses_tests
+      - run_android_services_fcm_tests
+      - run_android_services_http_tests
+      - run_android_services_mongodb_local_tests
+      - run_android_services_twilio_tests
+  - name: run_linter
+    setup_group_can_fail_task: true
+    max_hosts: 2
+    setup_group:
+      - func: "fetch_source"
+      - func: "setup_android"
+    tasks:
+      - findbugs
+      - lint
+      - ktlint
+      - checkstyle
+      - pmd
 buildvariants:
-- name: macos-1014-stitch
-  display_name: macos-1014-stitch
+- name: macos-1014
```

Comment:
[q] why not macos-1014-stitch? If the server team ever gets hit with a bunch of builds all at once, this could delay our task significantly

Reply:
There's only one macos-1014-stitch box. There are many macos-1014 boxes now. It allows us to parallelize our tasks (which is why the build time is so short now)

Reply:
right, I guess what I was trying to say is that sometimes the server time will queue thousands of tasks, meaning that all of the macos-1014 boxes will have a potentially much longer queue than macos-1014-stitch. But I guess in general this is better, so we can definitely keep it